where-you-can-find/NoteBook.HTML

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>Jaoh</h1>
+    </header>
+    <h2>Contents</h2>
```

Comment:
```suggestion
    <h2>Contents</h2>
```
    <nav><a href="#">About Me</a></nav>
    <h2>Contents</h2>